mne_bids_pipeline/scripts/preprocessing/_06_ptp_reject.py

```diff
@@ -121,9 +135,11 @@ def get_config(
         reject_tmin=config.reject_tmin,
         reject_tmax=config.reject_tmax,
         spatial_filter=config.spatial_filter,
-        ica_reject=config.get_ica_reject(),
+        ica_reject=config.ica_reject,
         deriv_root=config.get_deriv_root(),
         decim=config.decim,
+        reject=config.reject,
```

Comment:
And perhaps the worst offender, where changing `reject` wouldn't be detected by our caching mechanisms because it was silently used inside a config function call rather than being explicitly passed to a function isolated from config (where variables must be passed explicitly)